js/custom_projection.js

```diff
@@ -176,8 +176,8 @@ function albersUsaTerritories() {
         .stream(pointStream);
 
     puertoRicoPoint = puertoRico
-        .translate([x + 0.350 * k, y + 0.224 * k])
-        .clipExtent([[x + 0.312 * k + epsilon, y + 0.2064 * k + epsilon],[x + 0.385 * k - epsilon, y + 0.233 * k - epsilon]])
+        .translate([x + 0.340 * k, y + 0.170 * k])
+        .clipExtent([[x + 0.280 * k + epsilon, y + 0.130 * k + epsilon],[x + 0.410 * k - epsilon, y + 0.190 * k - epsilon]])
```

Comment:
cool, nice work figuring this out! do you think https://github.com/USGS-VIZLAB/water-use-15/pull/101/files#diff-1b045ffa0e22d4eccda9408d9c5d9777R81 might also need to be updated to match these changes?

Reply:
So I'm confused on this the link just leads me to what I just changed.  Did you mean to do that?

Reply:
Never mind, needed to scroll up

Reply:
I wouldn't change those, I'm not exactly sure but I feel those numbers have a higher purpose of how PR is projected, and the numbers I'm changing are just where PR is projected and how much of it is projected.  

The problem right now there is some uncertainty about what all these numbers do for the projection.  I only honestly feel comfortable making changes to the ones I did.

May not be an answer you find suitable, but its what I got at the moment.  Definitely, more study needed in projections.

Reply:
I believe the numbers I highlighted would be used if you wanted to be able to capture a set of svg coordinates (e.g., from a user's click) and turn that back into latitude and longitude (https://github.com/d3/d3-3.x-api-reference/blob/master/Geo-Projections.md#invert). That functionality is not important to water-use-15 functionality today but could be important to this or another viz in the future.

Playing around in the console:
```
// i think this is on alaska in the svg, goes to alaska in lon/lat
projection.invert([150,550])
(2) [-150.63359833273702, 64.08072856727983]
// https://www.google.com/maps/place/60%C2%B055'41.7%22N+149%C2%B007'00.5%22W/@63.5931454,-159.7661388,4.6z/data=!4m5!3m4!1s0x0:0x0!8m2!3d60.9282557!4d-149.1168018?hl=en

// just changing the y coordinate, but note that the x (and y) coords change a lot, so we're not in AK anymore
projection.invert([150,450])
(2) [-116.53973890285776, 32.118858217819]
```

Having looked a little more, I'm feeling increasingly confident that those numbers i highlighted should be adjusted to match your new numbers down here; also see how both alaska and hawaii match up almost perfectly? So I'm pushing a little harder - I advocate for changing those numbers now so they don't trip us up in the future, if and when we need to use `invert`.

Reply:
Alrighty, I'll trust you

Reply:
Trust now, assign blame later, whatever you need to do =)